nutaudet/Target.cxx

```diff
@@ -398,12 +388,15 @@ void Target::ConstructGeometry()
    volLead->SetTransparency(1);
    volLead->SetLineColor(kGray);
    //volLead->SetField(magField2);
+   volTungsten = NULL; //fixes -Wmaybe-uninitialized warning
```

Comment:
Maybe better to have a volume `absorber`, which then is either tungsten or lead depending on the config?

Then the other volume is not needed.

Reply:
Ok, I did not change however the volume names, so we keep the geometry files consistent.